style.css

```diff
@@ -1,5 +1,16 @@
 canvas {
     display: block;
     margin: auto;
-    border: 1px solid black;
+    border: 1px solid black;}
+
+div.container p {
+    font-family: Arial;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: bold;
+    text-decoration: none;
+    text-transform: none;
+    color: #000000;
+    background-color: #ffffff;
 }
+
```

Comment:
Since you're making CSS changes, a screenshot will assist in the code review. Could you please provide one?